webrtc/videoroom/priv/static/js/room.js

```diff
@@ -0,0 +1,231 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
This should be in .gitignore. Please check how does `.gitignore` look like in other examples

Reply:
Hmm they are looking the same